src/main/java/com/google/sps/storage/CaptionStorageException.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Wrapper exception class to contain all our exceptions thrown in CaptionStorage.java
+ */
+
+package com.google.sps.storage;
+
+public class CaptionStorageException extends Exception {
+
+    private Reason m_reason;
+
+    public CaptionStorageException(Reason r, String errorMessage, Throwable err) {
```

Comment:
Add method headers here (and all other methods in this class)

Reply:
think this is necessary even though they're just constructors and gets/sets?